### ISSUE TITLE
googleapis_common_protos EGG-INFO lacks o+r permission

### ISSUE THREAD

**Issue by lrowe** [googleapis/google-cloud-python#7819]
When installed as an egg, googleapis_common_protos EGG-INFO files lack world read permission. Example when installed with buildout:
```
$ find . ! -perm -o+r
./eggs/googleapis_common_protos-1.5.9-py3.7.egg/EGG-INFO/top_level.txt
./eggs/googleapis_common_protos-1.5.9-py3.7.egg/EGG-INFO/dependency_links.txt
./eggs/googleapis_common_protos-1.5.9-py3.7.egg/EGG-INFO/SOURCES.txt
./eggs/googleapis_common_protos-1.5.9-py3.7.egg/EGG-INFO/requires.txt
./eggs/googleapis_common_protos-1.5.9-py3.7.egg/EGG-INFO/namespace_packages.txt
./eggs/googleapis_common_protos-1.5.9-py3.7.egg/EGG-INFO/PKG-INFO
```

I'm then unable to import any namespaced package, for instance:
```
>>> import google.storage
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/srv/example/eggs/google_cloud_storage-1.14.0-py3.7.egg/google/__init__.py", line 2, in <module>
    __import__('pkg_resources').declare_namespace(__name__)
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3191, in <module>
    @_call_aside
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3175, in _call_aside
    f(*args, **kwargs)
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3219, in _initialize_master_working_set
    for dist in working_set
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3219, in <genexpr>
    for dist in working_set
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2724, in activate
    for pkg in self._get_metadata('namespace_packages.txt'):
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2714, in _get_metadata
    for line in self.get_metadata_lines(name):
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1416, in get_metadata_lines
    return yield_lines(self.get_metadata(name))
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1412, in get_metadata
    value = self._get(self._fn(self.egg_info, name))
  File "/srv/example/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1587, in _get
    with open(path, 'rb') as stream:
PermissionError: [Errno 13] Permission denied: '/srv/example/eggs/googleapis_common_protos-1.5.9-py3.7.egg/EGG-INFO/namespace_packages.txt'
```

I'm currently fixing this by running `find . ! -perm -o+r -exec chmod o+r {} \;` after installing.

**Comment by tseaver** [comment]
@lrowe Thanks for the report!  That distribution isn't released from this repo:  I've pinged @lukesneeringer to fix / re-relase it to PyPI.

**Comment by tseaver** [comment]
@lrowe I can't reproduce by untarring the [1.5.9 release sdist](https://files.pythonhosted.org/packages/76/a6/903a1c1c1fd676660174a9be4ec777c9b547392f640b16a326182b8fe484/googleapis-common-protos-1.5.9.tar.gz), either as myself or as `root`.  *None* of the files unpack as `o+r`.  How are you installing it?

**Comment by lukesneeringer** [comment]
I was able to reproduce (my machine had a umask of 0027). I released a 1.5.10 using a umask of 0022, which appeared to solve the issue.
